app/javascript/stylesheets/lesson-content.css

```diff
@@ -24,33 +24,83 @@
   }
 
   .lesson-note {
-    @apply bg-gray-50 p-4 border-l-4 border-gold dark:bg-gray-700/40 dark:border-gold-700 rounded;
+    @apply bg-gray-50 p-4 my-4 border-l-4 border-gold dark:bg-gray-700/40 dark:border-gold-600 rounded;
   }
 
-  .lesson-note > h4::before {
+  .lesson-note > *:first-child::before {
     font-family: "Font Awesome 6 Free";
+    font-weight: bold;
     content: "\f249";
-    @apply mr-2 text-gold dark:text-gold-700;
+    @apply mr-2 text-gold dark:text-gold-600;
+  }
+
+  .lesson-note > *:only-child::before {
+    display: block;
+    font-family: "Font Awesome 6 Free";
+    font-size: 1.8rem;
+    font-weight: bold;
+    content: "\f249";
+    @apply mr-2 text-gold dark:text-gold-600;
   }
 
   .lesson-note--tip {
-    @apply border-green-600 dark:border-green-700;
+    @apply border-blue-600 dark:border-blue-500;
+  }
+
+  .lesson-note--tip > *:first-child::before {
+    font-family: "Font Awesome 6 Free";
+    font-weight: bold;
+    content: "\f0eb";
+    @apply mr-2 text-blue-600 dark:text-blue-500;
   }
 
-  .lesson-note--tip > h4::before {
+  .lesson-note--tip > *:only-child::before {
+    display: block;
     font-family: "Font Awesome 6 Free";
+    font-size: 1.8rem;
+    font-weight: bold;
     content: "\f0eb";
-    @apply mr-2 text-green-600 dark:text-green-700;
+    @apply mr-2 text-blue-600 dark:text-blue-500;
   }
 
   .lesson-note--warning {
-    @apply border-red-700 dark:border-red-500;
+    @apply border-orange-700 dark:border-orange-600;
```

Comment:
I feel like these shades of orange are a little too dark when compared to the critical color... What do you think about `@apply border-orange-500 dark:border-orange-400;` instead?